go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/stretchr/testify v1.8.1
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
-	google.golang.org/protobuf v1.28.2-0.20230222093303-bc1253ad3743
+	google.golang.org/protobuf v1.28.2-0.20230306073304-fcf5f6cb7297
```

Comment:
This is to pick up a bug fix I contributed because the `prototext` implementation would not parse the syntax of aggregate values in uninterpreted options. This is because `protoc` always inserts a space between each lexical element when generating this value, which means it puts a space between a minus sign and the subsequent numeric value in a negative numeric literal. And `prototext` didn't allow such a space until this fix: https://github.com/protocolbuffers/protobuf-go/commit/fcf5f6cb72978812ac36085183b5c52701c069cb